packages/sources/layer2-sequencer-health/src/network.ts

```diff
@@ -45,7 +45,18 @@ export const checkSequencerHealth: NetworkHealthCheck = async (
   const response = await Requester.request({
     url: HEALTH_ENDPOINTS[network]?.endpoint,
   })
-  const isHealthy = !!Requester.getResult(response.data, HEALTH_ENDPOINTS[network]?.responsePath)
+
+  let isHealthy: boolean
+
+  // If the network has a custom response processing function, use it
+  if (HEALTH_ENDPOINTS[network]?.processResponse) {
```

Comment:
Instead of this condition can we create a defaultProcessResponse and use it for all other chains?

Reply:
you read my mind, thats exactly what i'm just about to push